bitrise.yml

```diff
@@ -167,7 +167,6 @@ workflows:
         - reporter: emoji
     - swiftlint@0.8.0:
         title: Lint Tests/*
-        is_always_run: true
```

Comment:
convenience: Do not lint `Tests/*` if linting `Sources/*` has failed. This is to avoid overwriting `swiftlint_report.txt` job artifact and keep the first one that reports the failure.